From 788a973a7635658212bd541a64cbea66499e9670 Mon Sep 17 00:00:00 2001
From: jye2 <jye2@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Sun, 4 May 2014 07:05:48 +0000
Subject: [PATCH] 2014-05-03  Joey Ye  <joey.ye@arm.com>

        Backport from mainline r209463
        2014-04-17  Joey Ye  <joey.ye@arm.com>

        * opts.c (OPT_fif_conversion, OPT_fif_conversion2): Disable for Og.


git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_9-branch@210049 138bc75d-0d04-0410-961f-82ee72b054a4
---
 gcc/opts.c    |    4 ++--
 2 files changed, 9 insertions(+), 2 deletions(-)

diff --git a/gcc/opts.c b/gcc/opts.c
index fdc903f..3f3db1a 100644
--- a/gcc/opts.c
+++ b/gcc/opts.c
@@ -431,8 +431,8 @@ static const struct default_options default_options_table[] =
     { OPT_LEVELS_1_PLUS, OPT_fguess_branch_probability, NULL, 1 },
     { OPT_LEVELS_1_PLUS, OPT_fcprop_registers, NULL, 1 },
     { OPT_LEVELS_1_PLUS, OPT_fforward_propagate, NULL, 1 },
-    { OPT_LEVELS_1_PLUS, OPT_fif_conversion, NULL, 1 },
-    { OPT_LEVELS_1_PLUS, OPT_fif_conversion2, NULL, 1 },
+    { OPT_LEVELS_1_PLUS_NOT_DEBUG, OPT_fif_conversion, NULL, 1 },
+    { OPT_LEVELS_1_PLUS_NOT_DEBUG, OPT_fif_conversion2, NULL, 1 },
     { OPT_LEVELS_1_PLUS, OPT_fipa_pure_const, NULL, 1 },
     { OPT_LEVELS_1_PLUS, OPT_fipa_reference, NULL, 1 },
     { OPT_LEVELS_1_PLUS, OPT_fipa_profile, NULL, 1 },
-- 
1.7.1

